packages/components/NativeRadioButton/macos/RadioButtonViewManager.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+@objc(RadioButtonViewManager)
+class RadioButtonViewManager: RCTViewManager {
+
+	override func view()->NSView! {
+		let radioButton = RadioButton.init(radioButtonWithTitle: "", target: nil, action: nil)
+		radioButton.action = #selector(radioButton.callOnPress)
```

Comment:
we can't combine this directly in line 7?

Reply:
I think radioButton needs to be initialized before I can access it as an object.

Reply:
am I misunderstanding something? it can't be target self action #selector(callonpress) ?

Reply:
Target must be a NSButton object and `self` here is RCTViewManger so I had to create an instance of RadioButton and set itself as the target. `callOnPress` belongs to the RadioButton class, so it needs to be accessed via a RadioButton object. 

I couldn't get init radioButtonWithTitle working in RadioButton so I came up with this workaround instead.